examples/common/olm-subscriptions/README.md

```diff
@@ -0,0 +1,13 @@
+# OLM subscription that are used for deploying specific versions.
+
+To deploy a specific version rather than the latest from OLM images
+containing multiple version definitions, it is necessary to explicitly
+set the `startingCSV` parameter.
+
+This kustomization setup accomplishes this for:
+- v1.0.3
+- v1.0.6: this version no longer includes the AnsibleEE operator
+- v1.0.7 and later: these versions only include the OpenStack operator
+
+Refer to the `ci_gen_kustomize_values` role README for configuration
```

Comment:
It would be good to provide a link to CIFMW if we're going to reference it, though I am a bit wary of pointing in that direction since the dependency is the other way around and we haven't explicitly mentioned CIFMW in this repo yet.

Reply:
Eventually `ci_gen_kustomize_values`  is one "hardcoded" way to interact with this (as it is for `examples/common/olm/`) and it took me some time to get it.  From a roocky point of view it makes sense to explicititely set the relation between the two, but I might miss something. Now, I get that maybe such pointer shouldn't be explicit in there.  So let me know when you've set your mind, and I'll either add a proper link, or just remove that line entirely.